### PR TITLE
Bid adieu to CLUSTER_IP_RANGE

### PR DESCRIPTION
**Pull request by shyamjvs**
Ref https://github.com/kubernetes/test-infra/issues/3375

Removed one more env var :)

/cc @fejta @krzyzacy

### CODE REVIEW

**Comment by krzyzacy** [comment]
weee, nice!

/lgtm

**Comment by shyamjvs** [comment]
Removed the flags, but forgot to remove the env var from job configs. Done now.

**Comment by shyamjvs** [comment]
@kubernetes/test-infra-maintainers I've noticed it on more than one occasion that lgtm in this repo doesn't get cancelled if changes are made after the lgtm.

**Comment by krzyzacy** [comment]
there's no munger here to cancel the lgtm label

**Comment by shyamjvs** [comment]
Why not?

**Comment by fejta** [comment]
Awesome!

File: jobs/config.json [hunk 1]
```diff
@@ -2539,7 +2539,7 @@
       "--gcp-zone=us-central1-b",
       "--mode=docker",
       "--provider=gce",
-      "--test_args=--ginkgo.focus=\\[Feature:Empty\\] --allowed-not-ready-nodes=10 --node-schedulable-timeout=60m --cluster-ip-range=10.160.0.0/13",
```
Comment (fejta) [inline jobs/config.json]: Shouldn't we delete the --cluster-ip-range flag?

**Comment by krzyzacy** [comment]
mungers/submit-queue is not deployed in this repo - we are probably going to rewrite the lgtm cancel behavior as a prow plugin instead.

File: jobs/config.json (hunk 1 above)
Comment (shyamjvs) [reply jobs/config.json]: Wanted to let it still stay in order to allow passing custom ranges.

**Pull request merged**